### ISSUE TITLE
UI/UX regression: Missing tbl_group link in navigation tree for groups

### ISSUE THREAD

**Issue by ETNyx** [phpmyadmin/phpmyadmin#17618]
If tables has a prefix like with double _ in name (like prefix__name) it makes a group. In older version (4.6.6 for sure) we were able to click on this group and filter tables by this group prefix.

Now in 5.2.0 there is no link. But necessary parameter still work "/index.php?route=/database/structure&db=database&tbl_group=prefix"

### To Reproduce
1. Make prefixed tables to show in groups
2. Click on group name
3. There is no link anymore.

### Expected behavior
Have a link whit "tbl_group=prefix" on group name to filter shown tables

### Screenshots
![Screenshot from 2022-06-28 12-58-49](https://user-images.githubusercontent.com/2624462/176163510-5fcaf94f-4340-460f-94e0-edac32f4e156.jpg)

### Server configuration
- Operating system: Linux
- Web server: Apache
- Database version: MariaDB 10.8.3
- PHP version: 8.0
- phpMyAdmin version: 5.2.0

### Client configuration
- Browser: Firefox
- Operating system: Linux

**Comment by williamdes** [comment]
Thank you for the report @ETNyx

**Comment by ETNyx** [comment]
@williamdes Hello, thanks, I'm glad someone pick it up. It will make my time in phpmyadmin much easier, thank you for that.

**Comment by pravintargaryen** [comment]
Guys is it still problem still open for contribution?

**Comment by pravintargaryen** [comment]
Hey Guys,

Is this issue still open for contributions?

**Comment by williamdes** [comment]
> Hey Guys,
> 
> Is this issue still open for contributions?

Yes, sure. Please use QA_5_2 as a base branch for your new branch and as a target for your PR
It's normally a bug and should be investigated why the link disappeared and if possible on what version :)

**Comment by deg026** [comment]
probable fast fix here (duplicate issue):
https://github.com/phpmyadmin/phpmyadmin/issues/17947
